snooty/main.py

```diff
@@ -48,6 +49,7 @@
 logger = logging.getLogger(__name__)
 
 EXIT_STATUS_ERROR_DIAGNOSTICS = 2
+EXT_FOR_PAGE = ".txt"
```

Comment:
We have this hard-coded in a lot of different spots

Could we actually move this to `util.py` so other files can use it?

Reply:
done! replaced a few places in postprocess which references this